### PR TITLE
WIP(KYC): add terms and conditions dialog

### PR DESCRIPTION
**Pull request by 0oM4R**
### Description

Add TC dialog before the KYC 
image
### Changes

- added a TC dialog, and block the verification process.

### Related Issues

- #3599

### Tested Scenarios

- closed the dialog without agree
- the verification dialog should appear only after agree and click on continue

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by xmonader — APPROVED** [review]

**Comment by ramezsaeed** [comment]
for Tested Scenarios we can add:
- closed the dialog without agree.
- the verification dialog should appear only after agree and click on continue.
- refresh the page after check the box and before press continue.
- continue button should only be visible if the checkbox is checked.

**Review by ramezsaeed — APPROVED** [review]
for Tested Scenarios we can add:
- closed the dialog without agree.
- the verification dialog should appear only after agree and click on continue.
- refresh the page after check the box and before press continue.
- continue button should only be visible if the checkbox is checked.
- check the link for terms and conditions is actually work.
- test verify button working fine from deploy page.

Testing blocked on: https://github.com/threefoldtech/tf_operations/issues/2902

File: packages/playground/src/components/KycVerifier.vue [hunk 1]
```diff
@@ -1,19 +1,55 @@
...
+
+        <v-card-text class="pb-0">
+          As part of the verification process, we utilize Idenfy to verify your identity.
+          <br />
+          Please ensure you review Idenfy’s <span class="font-weight-bold">Security and Compliance</span>, which include
+          their <span class="font-weight-bold">Terms & Conditions, Privacy Policy</span>, and other relevant documents.
+          <v-checkbox hide-details v-model="agreedCheckbox">
+            <template v-slot:label>
+              <div>
+                I have read and agreed to
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue:38]:
```suggestion
                I have read and agreed to Idefy Terms & Conditions
```

File: packages/playground/src/components/KycVerifier.vue [hunk 2]
```diff
@@ -1,19 +1,55 @@
...
+
+                <a href="https://www.idenfy.com/security/" target="_blank" @click.stop>
+                  IDenfy Security and Compliance</a
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]:
```suggestion
                  iDenfy Terms & Conditions</a
```

File: packages/playground/src/components/KycVerifier.vue [hunk 3]
```diff
@@ -1,19 +1,55 @@
...
+      <v-card>
+        <v-card-title class="bg-primary d-flex align-center">
+          <v-icon icon="mdi-security" />
+          <div class="pl-2">Terms and conditions</div>
+        </v-card-title>
+
+        <v-card-text class="pb-0">
+          As part of the verification process, we utilize Idenfy to verify your identity.
+          <br />
+          Please ensure you review Idenfy’s <span class="font-weight-bold">Security and Compliance</span>, which include
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]:
```suggestion
          Please ensure you review iDenfy’s <span class="font-weight-bold">Security and Compliance</span>, which include
```

File: packages/playground/src/components/KycVerifier.vue [hunk 4]
```diff
@@ -1,19 +1,55 @@
...
+      max-width="700"
+    >
+      <v-card>
+        <v-card-title class="bg-primary d-flex align-center">
+          <v-icon icon="mdi-security" />
+          <div class="pl-2">Terms and conditions</div>
+        </v-card-title>
+
+        <v-card-text class="pb-0">
+          As part of the verification process, we utilize Idenfy to verify your identity.
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]:
```suggestion
          We use iDenfy to verify your identity.
```

File: packages/playground/src/components/KycVerifier.vue [hunk 5]
```diff
@@ -1,19 +1,55 @@
...
+    </v-dialog>
+    <v-dialog
+      v-model="agreementDialog"
+      @update:model-value="($event: boolean) => !$event ? handleUpdateDialog($event) : undefined"
+      max-width="700"
+    >
+      <v-card>
+        <v-card-title class="bg-primary d-flex align-center">
+          <v-icon icon="mdi-security" />
+          <div class="pl-2">Terms and conditions</div>
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]:
```suggestion
          <div class="pl-2">Terms & Conditions</div>
```

Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]: check padding design with @ehab-hassan i think it can use more padding, no?

**Comment by amiraabouhadid** [comment]
i get this when i press continue . is this error tracked? 
image
everytime i press verify afterwards i get the same error
even after creating another account

**Comment by AhmedHanafy725** [comment]
> i get this when i press continue . is this error tracked? image everytime i press verify afterwards i get the same error even after creating another account

okay it's not related to the main purpose of the PR

File: packages/playground/src/components/KycVerifier.vue [hunk 6]
```diff
@@ -1,19 +1,53 @@
...
+      <v-card>
+        <v-card-title class="bg-primary d-flex align-center">
+          <v-icon icon="mdi-security" />
+          <div class="pl-2">Terms & Conditions</div>
+        </v-card-title>
+
+        <v-card-text class="pb-0">
+          We use iDenfy to verify your identity.
+          <br />
+          Please ensure you review iDenfy’s <span class="font-weight-bold">Security and Compliance</span>, which include
```
Comment (amiraabouhadid) [inline packages/playground/src/components/KycVerifier.vue]:
```suggestion
          Please ensure you review iDenfy’s <span class="font-weight-bold">Security and Compliance</span>, which includes
```

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**